editor/src/uuiui/inputs/string-input.tsx

```diff
@@ -77,6 +78,46 @@ export const StringInput = React.memo(
 
       const placeholder = getControlStylesAwarePlaceholder(controlStyles) ?? initialPlaceHolder
 
+      const onChange = React.useCallback(
```

Comment:
btw, would we expect this logic (`onChange`, `onPaste`) to be on the StringInput or the HeadlessStringInput?